### PR TITLE
Fixed everything terrible about this page

### PR DESCRIPTION
**Pull request by pseudoname23**
The formatting and indentation was awful, the code was outdated and cobbled together from about 3 different websites (I saw snippets from W3schools, MDN Web Docs, and metatags.io), there were HTML elements in the head, and the script was between the head and the body. You could tell that whoever wrote this clusterfuck originally had no idea what they were doing.

### CODE REVIEW

**Comment by smokeytube** [comment]
> The formatting and indentation was awful, the code was outdated and cobbled together from about 3 different websites (I saw snippets from W3schools, MDN Web Docs, and metatags.io), there were HTML elements in the head, and the script was between the head and the body. You could tell that whoever wrote this clusterfuck originally had no idea what they were doing.

i sory 😭😭😭😭😭

**Comment by SoNothingMC** [comment]
@pseudoname23 Do you accept this guy's apology

**Comment by EmeraldSnorlax** [comment]
image

**Review by EmeraldSnorlax — APPROVED** [review]

File: printer/javascript.txt [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
-at least it's not lisp though. (((((help)))))
-everyone hates rust because it's the oh so cool new thing to do and it's so much better, the ultimate language to rule them all and every last piece of software absolutely must be rewritten in rust but, honestly, the borrow checker keeps me from getting anything done. also compile times duh.
-everyone hates D because ... wait, nobody hates D because nobody uses it. also, why settle on D if you could use rust? D is for people who can't escape the sunk cost fallacy.
-everyone hates golang because no generics and you're pretty much at googles mercy if you specialize in it, and we've seen how well that worked out for android devs.
-everyone hates swift and kotlin for not bringing anything new to the table. they maybe improve the worst problems (of java and objective-c) a bit, but in the end, they do nothing other languages haven't done better decades ago. pretty much the only revolutionarily good thing swift did was killing off objective-c. in the land of the blind, the one eyed is kind. java devs love kotlin only because at least it's not java.
```
Comment (XWasHere) [inline printer/javascript.txt:16]: > java devs love kotlin because at least it's not java.
not wrong though

Comment (smokeytube) [reply printer/javascript.txt:16]: D is only good for malware at this point tbh

**Comment by pseudoname23** [comment]
> @pseudoname23 Do you accept this guy's apology

yes

**Comment by JessFairbairn** [comment]
so much discord. I hope we can all maintain decorum and mutual respect in our shared goals. The batratmonster project is too important

**Pull request merged**